src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -51,9 +53,13 @@ protected final void onStartup() {
 
         // Need to discover the right codec for the driving source of variants manually, since we are
         // treating it specially (separate from the other sources of Features in our FeatureManager).
-        FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingVariantFile);
-        if (codec.getFeatureType() == VariantContext.class) {
+        final FeatureCodec<? extends Feature, ?> codec = FeatureManager.getCodecForFile(drivingVariantFile);
+        if (VariantContext.class.equals(codec.getFeatureType())) {
             drivingVariants = new FeatureDataSource<>(drivingVariantFile, (FeatureCodec<VariantContext, ?>)codec);
+
+            //Add the driving datasource to the feature manager too so that it can be queried. Setting lookahead to 0 to avoid caching.
```

Comment:
Mention why we are disabling lookahead here (because of windowed queries that need to "look behind" as well)